frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```
@@ -532,8 +532,9 @@
   "Pool {name} was successfully created": "Pool {name} was successfully created",
   "An error occurred Pool {name} was not created": "An error occurred Pool {name} was not created",
   "Pool {name} creation timed out. Please check if ocs-operator and rook operator are running": "Pool {name} creation timed out. Please check if ocs-operator and rook operator are running",
-  "The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.": "The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.",
-  "Pool management tasks are not supported for default pool and Openshift Container Storage's  external mode.": "Pool management tasks are not supported for default pool and Openshift Container Storage's  external mode.",
+  "Pool {name} got created with errors": "Pool {name} got created with errors",
+  "Pool management tasks are not supported for default pool and Openshift Container Storage's external mode.": "Pool management tasks are not supported for default pool and Openshift Container Storage's external mode.",
+  "The creation of an OCS StorageCluster is still in progress or has failed. Please try again after the StorageCluster is ready to use.": "The creation of an OCS StorageCluster is still in progress or has failed. Please try again after the StorageCluster is ready to use.",
```

The usage of OCS should be avoided ,  use full product name wherever required.
Here it can be avoided.
```suggestion
  "The creation of a StorageCluster is still in progress or has failed. Please try again after the StorageCluster is ready to use.": "The creation of an OCS StorageCluster is still in progress or has failed. Please try again after the StorageCluster is ready to use.",
```

---

ack